ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -588,16 +604,17 @@ void handleResponse(ResponseInfo responseInfo) {
       }
       long requestLatencyMs = time.milliseconds() - getRequestInfo.startTimeMs;
       routerMetrics.routerRequestLatencyMs.update(requestLatencyMs);
-      routerMetrics.getDataNodeBasedMetrics(getRequestInfo.replicaId.getDataNodeId()).getRequestLatencyMs
-          .update(requestLatencyMs);
+      routerMetrics.getDataNodeBasedMetrics(getRequestInfo.replicaId.getDataNodeId()).getRequestLatencyMs.update(
```

Comment:
We don't seem to be using the same code formatting templates. I think these lines keep getting switched back and forth.